build.gradle

```
@@ -69,6 +76,7 @@ repositories {
 dependencies {
     implementation 'Red-Studio-Ragnarok:Red-Core:0.4-Dev-2'
     sources 'Red-Studio-Ragnarok:Red-Core:0.4-Dev-2:sources@jar'
+    compileOnly rfg.deobf("curse.maven:groovyscript-687577:4905039")
```

This should be replaced by properly depending on it
```suggestion
    implementation 'com.cleanroommc:groovyscript:0.7.2'
```